com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Constants.java

```diff
@@ -49,6 +49,7 @@
     public static final String PARAM_SKIP_OWNERSHIP_VALIDATION = "skipOwnershipValidation";
     public static final String PARAM_MODULES_FOR_DEPLOYMENT = "modulesForDeployment";
     public static final String PARAM_RESOURCES_FOR_DEPLOYMENT = "resourcesForDeployment";
+    public static final String PARAM_BUILD_GUID = "buildGuid";

```

Comment:
Rename this to `VAR_BUILD_GUID`. The `PARAM` prefix is only used for variables, which are given to the process during its startup.